src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -245,17 +251,12 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
           </ClaimTable>
 
           <AccountClaimSummary>
+            <AccountDetails index={0} label="Claiming with account" account={account} connectedAccount={account} />
```

Comment:
And you would only need to include the flag in the first component

```suggestion
            <AccountDetails isClaimer label="Claiming with account" account={account} connectedAccount={account} />
```

Reply:
@alfetopito Aaah much cleaner like that, thank you!